### PR TITLE
 Bump manifests for serving core to 1.3.0

### PR DESCRIPTION
**Pull request by skonto**
## Proposed Changes

- The usual bumping for Serving.
- Serving is actually on 1.3.2, compare commits [here](https://github.com/openshift/knative-serving/commits/release-v1.3) and [here](https://github.com/knative/serving/tree/knative-v1.3.2) but for images we use 1.3.0 as a fixed namespace. Also manifests havent changed between minor versions.
- In the future manifests will be fetched from the midstream repo instead of downloading from upstream, but let's stabilize for 1.24 before migrating to that.

### CODE REVIEW

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -430,33 +430,33 @@ spec:
                       - name: SERVICE_MONITOR_RBAC_MANIFEST_PATH
                         value: "/var/run/ko/monitoring/rbac-proxy.yaml"
                       - name: "IMAGE_queue-proxy"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.2.0:knative-serving-queue"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.3.2:knative-serving-queue"
```
Comment (skonto) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:433]: Images are not built yet.

Comment (matzew) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml]: @skonto they are versioned as `1.3.0` - see: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v1.3__410.yaml#L223

Comment (matzew) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml]: Ah, you forced pushed the commit and updated to 1.3.0 

yes, those are available and I can `docker pull` 'em

**Comment by nak3** [comment]
/lgtm
/hold

`/hold` for CI.

**Comment by skonto** [comment]
```
Failed to pull image "quay.io/openshift-knative/kafka-consumer:v1.3": rpc error: code = Unknown desc = reading manifest v1.3 in quay.io/openshift-knative/kafka-consumer: manifest unknown: manifest unknown
```
Let's poke CI for Eventing on that branch.

**Comment by skonto** [comment]
Image is now available. Retesting. /retest

**Comment by skonto** [comment]
Tests pass /unhold

**Pull request merged**